src/components/dataset-query/dataset-query.js

```diff
@@ -0,0 +1,31 @@
+import { useState, useEffect } from 'react'
+import './dataset-query.scss'
+
+const DatasetQuery = props => {
+  const defaultText = `SELECT * FROM ${props.systemName}\nLIMIT 20000`
```

Comment:
Why place the limit on a new line?

Reply:
Preference, trying to follow some sort of standard SQL styling.  Also makes it more obvious to the user IMO.